senior_teacher.rb

```diff
@@ -1,22 +1,18 @@
-class SeniorTeacher
-  attr_reader :age, :salary, :phase, :performance_rating, :target_raise
+require_relative 'staff'
+
+class SeniorTeacher < Staff
+  attr_reader :age, :salary,:phase, :performance_rating, :target_raise
   attr_accessor :name
 
   def initialize(options={})
+    super
     @phase = 3
-    @age = options.fetch(:age, 0)
-    @name = options.fetch(:name, "")
+    # @age = options.fetch(:age, 0)
+    # @name = options.fetch(:name, "")
     @target_raise = 1000
   end
 
-  def offer_high_five
-    "High five!"
-  end
-
-  def set_phase(num)
-    @phase = num
-    "Cool, I've always wanted to teach phase #{num}!"
-  end
+  RATING = 90
```

Comment:
Also, the name for this should probably be TARGET_RATING to make its role clear.